jobbergateapi2/apps/applications/routers.py

```diff
@@ -89,36 +79,33 @@ async def application_delete(
     raw_application = await database.fetch_one(get_query)
     if not raw_application:
         raise HTTPException(
-            status_code=status.HTTP_404_NOT_FOUND,
-            detail=f"Application {application_id=} not found.",
+            status_code=status.HTTP_404_NOT_FOUND, detail=f"Application {application_id=} not found.",
         )
     application = Application.parse_obj(raw_application)
     delete_query = applications_table.delete().where(where_stmt)
     await database.execute(delete_query)
-    application_location = f"{settings.S3_BASE_PATH}/{application.application_owner_id}/applications/{application.id}/jobbergate.tar.gz"  # noqa
+    application_location = "{base}/{owner_id}/applications/{app_id}/jobbergate.tar.gz".format(
+        base=settings.S3_BASE_PATH, owner_id=application.application_owner_id, app_id=application_id,
+    )
```

Comment:
Can we make a little helper out of this since we use it in > 1 place?

Reply:
Created a manager class for the s3 client that wraps all the s3 interactions up in a nice little interface. See latest commit...